amplify/function/utils/crytography.ts

```diff
@@ -0,0 +1,39 @@
+import { kmac128 } from "js-sha3";
+
+//Functions to help with the creation and verification of a Message Authentication Code
+export async function isValidAuthenticationCode(
```

Comment:
Do we really need this level of security? I understand the QR Code can be tampered with, but the chances are extremely low. It's really cool that you added this mechanism, but we want to make it as simple as possible for future CTC members to maintain. If you want to keep this code, could you make a case for it?

Reply:
replied on slack, but copied here for others:

I do agree that it is overkill. But, I remember it was stated that we wanted things to be secure from the start since the hackathoners know tech really well and we should be careful, (although technically guessing a UUID is really hard still). Also, this user code is not just for food tickets, but for checking the user ID as a whole, it was just that we used it for food tickets here. Also, if you are worried about the readability of this, I did write documentation for it in the README, and this is best practice (which I thought was the point of this project), and will be a good teaching experience for the future devs (I did write a lot of comments to help with this too)